mmrazor/models/losses/l2_loss.py

```diff
@@ -49,10 +53,14 @@ def forward(
 
         loss = torch.sum(torch.pow(torch.sub(s_feature, t_feature), 2))
 
-        if self.div_element:
-            loss = loss / s_feature.numel()
+        # Calculate l2_loss as dist.
+        if self.dist:
+            loss = torch.sqrt(loss)
         else:
-            loss = loss / s_feature.size(0)
+            if self.div_element:
+                loss = loss / s_feature.numel()
+            else:
+                loss = loss / s_feature.size(0)
```

Comment:
For line67-75, whether the mean should be subtracted during normalization. If so, can we replace it with `F.layer_norm()`

Reply:
In gml source code mean is not subtracted, and only conduct on channel.